src/components/editions/headerImage.tsx

```diff
@@ -55,19 +55,25 @@ const captionStyles = css`
 `;
 
 const fullWidthCaptionStyles = css`
-	width: 100vw;
+	width: 100%;
 
-	${from.wide} {
-		width: 100vw;
+	${from.tablet} {
+		width: 100%;
```

Comment:
The header image was overflowing with 100vw as it included spacings/ this also shifted the camera icon.

Reply:
Nice, I'd seen that popping up last week.